lib/evss/disability_compensation_form/data_translation.rb

```diff
@@ -0,0 +1,236 @@
+            'lastName' => an['last']
+          }.compact
+        end
+      end
+
+      def convert_national_guard_service(reserves_service_info)
+        {
+          'title10Activation' => reserves_service_info['title10Activation'],
+          'obligationTermOfServiceFromDate' => reserves_service_info['obligationTermOfServiceDateRange']['from'],
+          'obligationTermOfServiceToDate' => reserves_service_info['obligationTermOfServiceDateRange']['to'],
+          'unitName' => reserves_service_info['unitName'],
+          'unitPhone' => split_phone_number(reserves_service_info['unitPhone']),
+          'inactiveDutyTrainingPay' => reserves_service_info['inactiveDutyTrainingPay']
+        }.compact
+      end
+
+      def convert_veteran
+        veteran['primaryPhone'] = split_phone_number(veteran['phone'])
+        veteran.delete('phone')
+
+        veteran['mailingAddress'] = convert_mailing_address(veteran['mailingAddress'])
+        if veteran['forwardingAddress'].present?
+          veteran['forwardingAddress'] = convert_mailing_address(veteran['forwardingAddress'])
+        end
+
+        homeless_data = veteran['homelessness']
+        if homeless_data['isHomeless'].present?
+          veteran['homelessness'] = convert_homelessness(homeless_data['pointOfContact'])
+        else
+          veteran.delete('homelessness')
+        end
+      end
+
+      def convert_homelessness(point_of_contact)
+        homelessness = {}
+        if point_of_contact.present?
+          homelessness['hasPointOfContact'] = true
+          homelessness['pointOfContact'] = {
+            'pointOfContactName' => point_of_contact['pointOfContactName'],
+            'primaryPhone' => split_phone_number(point_of_contact['primaryPhone'])
+          }
+        else
+          homelessness['hasPointOfContact'] = false
+        end
+
+        homelessness
+      end
+
+      def service_branch(service_branch)
+        case service_branch
+        when 'NOAA'
+          'National Oceanic &amp; Atmospheric Administration'
+        else service_branch
+        end
+      end
+
+      def split_phone_number(phone_number)
+        area_code, number = phone_number.match(/(\d{3})(\d{7})/).captures
+        { 'areaCode' => area_code, 'phoneNumber' => number }
+      end
+
+      def get_banking_info
+        service = EVSS::PPIU::Service.new(@user)
+        response = service.get_payment_information
+        account = response.responses.first.payment_account
+
+        if account
+          {
+            'accountType' => account&.account_type&.upcase,
+            'accountNumber' => account&.account_number,
+            'routingNumber' => account&.financial_institution_routing_number,
+            'bankName' => account&.financial_institution_name
+          }
+        end
+      end
+
+      def convert_mailing_address(address)
+        pciu_address = { 'country' => address['country'],
+                         'addressLine1' => address['addressLine1'],
```

Comment:
The naming here was [discussed](https://github.com/department-of-veterans-affairs/vets-website/pull/7846/files#r194169990) recently, but I'm not sure whether there was consensus on if changing this was of interest.